Aws/DynamoDb/Core.hs

```diff
@@ -1127,6 +1138,7 @@ class DynSize a where
     dynSize :: a -> Int
 
 instance DynSize DValue where
+    dynSize DNull = 8
```

Comment:
I'm not sure where to get the sizes from. I'm guessing this would probably be the same as `DBool` but I'm confused as to if these are meant to be bytes, bits or something else (seems like maybe bits but then `DString` would be wrong)